src/plugins/saved_objects_management/public/plugin.ts

```diff
@@ -236,7 +236,11 @@ export class SavedObjectsManagementPlugin
             }),
         };
       },
-      getTargetArea: () => HOME_CONTENT_AREAS.RECENTLY_VIEWED,
+      getTargetArea: () => [
+        HOME_CONTENT_AREAS.RECENTLY_VIEWED,
+        'essential_overview/recently_viewed',
+        'analytics_all_overview/recently_viewed',
```

Comment:
I am thinking if we should put these ids in content management plugin.

Reply:
content management plugin could be a central place for all page ids, section id. @ruanyl any suggestions?

Reply:
@Hailong-am yes, I'm also thinking of moving these ids to content management plugin

Reply:
thanks, i will move all those ids into content management plugin